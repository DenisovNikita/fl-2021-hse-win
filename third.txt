0 to 1 by x
0 to 0 by y
0 to 0 by z
1 to 1 by x
1 to 1 by y
1 to 1 by z
terminal 1
1
