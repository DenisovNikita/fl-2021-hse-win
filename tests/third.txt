0 to 1 by 1 x
0 to 0 by 1 y
0 to 0 by 1 z
1 to 1 by 1 x
1 to 1 by 1 y
1 to 1 by 1 z
terminal 1
1
