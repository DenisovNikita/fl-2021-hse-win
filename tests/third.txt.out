LexToken(NUM,0,1,0)
LexToken(TO,'to',1,2)
LexToken(NUM,1,1,5)
LexToken(BY,'by',1,7)
LexToken(NUM,1,1,10)
LexToken(CHAR,'x',1,12)
LexToken(NUM,0,2,14)
LexToken(TO,'to',2,16)
LexToken(NUM,0,2,19)
LexToken(BY,'by',2,21)
LexToken(NUM,1,2,24)
LexToken(CHAR,'y',2,26)
LexToken(NUM,0,3,28)
LexToken(TO,'to',3,30)
LexToken(NUM,0,3,33)
LexToken(BY,'by',3,35)
LexToken(NUM,1,3,38)
LexToken(CHAR,'z',3,40)
LexToken(NUM,1,4,42)
LexToken(TO,'to',4,44)
LexToken(NUM,1,4,47)
LexToken(BY,'by',4,49)
LexToken(NUM,1,4,52)
LexToken(CHAR,'x',4,54)
LexToken(NUM,1,5,56)
LexToken(TO,'to',5,58)
LexToken(NUM,1,5,61)
LexToken(BY,'by',5,63)
LexToken(NUM,1,5,66)
LexToken(CHAR,'y',5,68)
LexToken(NUM,1,6,70)
LexToken(TO,'to',6,72)
LexToken(NUM,1,6,75)
LexToken(BY,'by',6,77)
LexToken(NUM,1,6,80)
LexToken(CHAR,'z',6,82)
LexToken(TERMINAL,'terminal',7,84)
LexToken(NUM,1,7,93)
LexToken(NUM,1,8,95)
